src/components/portal/__spec__.js

```diff
@@ -1,27 +1,74 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { mount, shallow } from 'enzyme';
 import Portal from './portal';
-import ReactPortal from 'react-portal';
+import Browser from '../../utils/helpers/browser';
 
 describe('Portal', () => {
-  let child, wrapper, reactPortal;
-
-  beforeEach(() => {
-    wrapper = shallow(
-      <Portal open>
-        <p className='child-element'></p>
-      </Portal>
-    );
+  let wrapper;
+  describe('when using default node', () => {
+    beforeEach(() => {
+      wrapper = shallow(
+        <Portal>
+          <p className='child-element' />
+        </Portal>
+      );
+    });
+
+    it('div with name portal should be created ', () => {
+      expect(wrapper.instance().node.classList[0]).toEqual('portal');
+    });
+
+    it('to match snapshot ', () => {
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('will unmount correctly', () => {
+      expect(wrapper.unmount());
+    });
   });
 
-  it('renders an instance of ReactPortal', () => {
-    reactPortal = wrapper.find(ReactPortal);
-    expect(reactPortal.length).toEqual(1);
-    expect(reactPortal.props().isOpened).toBeTruthy();
+  describe('when using a custom node', () => {
+    beforeEach(() => {
+      const node = Browser.getDocument().createElement('div');
+      node.classList.add('my-node');
+
+      wrapper = mount(
+        <Portal node={ node }>
+          <p className='child-element' />
+        </Portal>
+      );
+    });
+
+    it('to match snapshot ', () => {
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('the node prop has been added to the DOM', () => {
+      expect(wrapper.instance().node.classList[0]).toEqual('my-node');
```

Comment:
not sure what this is testing other than it is assigning your custom element to the instance?

Maybe it should test that the portal has mounted the child components within your custom node?